core/project/Dependencies.scala

```diff
@@ -2,26 +2,28 @@ import sbt._
 
 // format: OFF
 object Version {
+
   val Akka         = "2.6.5"
   val AkkaHttp     = "10.1.11"
   val AkkaMgmt     = "1.0.6"
   val AlpakkaKafka = "2.0.3"
   val Scala        = "2.12.9"
```

Comment:
We can make this 2.12.11

Reply:
Good idea. Let me upgrade in a separate PR that we can test properly. I don't expect any issues but better to encapsulate that.

Reply:
Sure